packages/server/graphql/queries/suOrgCount.ts

```diff
@@ -35,29 +35,41 @@ export default {
     requireSU(authToken)
 
     // RESOLUTION
-    return r
-      .table('Organization')
-      .getAll(tier, {index: 'tier'})
-      .merge((organization) => ({
-        users: (r
-          .table('OrganizationUser')
-          .getAll(organization('id'), {index: 'orgId'})
-          .filter({removedAt: null})
-          .eqJoin('userId', r.table('User'))
-          .zip() as any)
-          .filter((user) =>
-            r.branch(
-              r(ignoreEmailRegex).eq(''),
-              true,
-              user('email')
-                .match(ignoreEmailRegex)
-                .eq(null)
-            )
-          )
-          .filter((user) => r.branch(includeInactive, true, user('inactive').not()))
-          .count()
-      }))
-      .filter((org) => org('users').ge(minOrgSize))
+    if (!ignoreEmailRegex && !includeInactive)
```

Comment:
I say let's just nuke support for ignoreEmailRegex and have a single query.

Reply:
@jordanh can we also nuke support for the option to include `inactive` users in the query?

Reply:
Done! Note that support for including inactive users in the query has also been nuked. Lmk if any objections.